presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1030,7 +1030,13 @@ public Void visitTopN(TopNNode node, Integer indent)
         {
             Iterable<String> keys = Iterables.transform(node.getOrderingScheme().getOrderBy(), input -> input + " " + node.getOrderingScheme().getOrdering(input));
 
-            print(indent, "- TopN[%s by (%s)] => [%s]", node.getCount(), Joiner.on(", ").join(keys), formatOutputs(node.getOutputSymbols()));
+            print(
```

Comment:
Nit: this looks funny. I'd keep the first argument on the same line, like the other usages in this file.

Reply:
>  Nit: this looks funny. 

i agree

>I'd keep the first argument on the same line

First? Or first and second? IMO the latter looks better in this context and is used in the file already. And we are departing from code style rule anyway.

>  like the other usages in this file.

I tried to be consistent, but there isn't just one way of formatting this in the file.

Reply:
First and second seem fine to me, since it’s both more readable and consistent with the other usages.

Reply:
done